bld/namelist_files/namelist_defaults_ctsm_tools.xml

```diff
@@ -62,7 +62,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <scripgriddata hgrid="10x10min" lmask="nomask"    
    >lnd/clm2/mappingdata/grids/SCRIPgrid_10x10min_nomask_c110228.nc</scripgriddata>
 <scripgriddata hgrid="3x3min"    lmask="nomask"    
-   >lnd/clm2/mappingdata/grids/SCRIPgrid_3x3min_nomask_c191014.nc</scripgriddata>
+   >lnd/clm2/mappingdata/grids/SCRIPgrid_3x3min_nomask_c191102.nc</scripgriddata>
```

Comment:
The old file was an incorrectly labeled 0.125-degree file, as first documented in this [comment](https://github.com/ESCOMP/CTSM/pull/823#issuecomment-549002997).

Reply:
This most recent file name corresponds to the "trusted" generic 3x3min nomask file generated by mkscripgrid.ncl.

Same is true of the 5x5min file a few lines up from here.